includes/Admin/Dashboard.php

```diff
@@ -433,6 +433,7 @@ public function get_dashboard_settings(): array {
 				'settings'       => '/web-stories/v1/settings/',
 				'pages'          => '/wp/v2/pages/',
 				'publisherLogos' => '/web-stories/v1/publisher-logos/',
+				'taxonomies'     => '/web-stories/v1/taxonomies/',
```

Comment:
Let's also preload this api, by adding to this area. 
https://github.com/GoogleForCreators/web-stories-wp/blob/3d0a8a1db3c3e4b6120219b183c73ab9d744885f/includes/Admin/Dashboard.php#L302